pyproject.toml

```diff
@@ -6,7 +6,7 @@ readme = "README.md"
 authors = [{name = "Sym Roe"}, {name = "Virginia Dooley"}]
 
 dependencies = [
-    "django>=3.2,<4.3",
+    "django>=4.2",
```

Comment:
In general, unless we know we're relying on something we know will break in a future release, I'd tend to avoid upper-bound version constraints for python packages.